### PR TITLE
v8 3.15 from homebrew/versions

### PR DESCRIPTION
**Pull request by msbit**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Aliases/v8@5.1 [hunk 1]
```diff
@@ -0,0 +1 @@
+../Formula/v8.rb
```
Comment (fxcoudert) [inline Aliases/v8@5.1]: Why name the alias `v8@5.1` and not `v8@5` which seems more consistent with our policy?

File: formula_renames.json [hunk 2]
```diff
@@ -143,5 +143,6 @@
   "stash-cli": "atlassian-cli",
   "tomcat6": "tomcat@6",
   "tomcat7": "tomcat@7",
+  "v8-315": "v8@3.15",
```
Comment (fxcoudert) [inline formula_renames.json]: Same here: shouldn't we rename it to `v8@3`?

**Comment by msbit** [comment]
@fxcoudert hmm, I'm not sure about the major/minor for libv8. The project seems to run at a crazy pace, with 1751 tags under `v4`, and 3793 and counting under `v5`. I think the importance of that specific version (`v3.15.11.18`) was that it was the latest version that was compatible with a particular Ruby gem version (and perhaps a major Ruby version itself). I definitely remember going through the pain of getting that working in the past.

I know it's not entirely in keeping with trying to avoid minor version formulae/aliases, but if it were to go to `v8@3` from a glance I'd expect that to be `v3.32.7`, the latest 3.x release, so I _feel_ like it suits a `v8@3.15`. 

I can accept the status quo, and I'm not the person to make the bigger picture decision about `v8` in Homebrew in general, so perhaps it's best to drop this and it's sister PR (https://github.com/Homebrew/homebrew-versions/pull/1548).

**Comment by msbit** [comment]
@fxcoudert closing this and related PR, as V8 support policy is for current and previous minor releases (5.7 and 5.8 at the moment) and a new minor is cut every 6 weeks. This makes this version unsupported so not really a good candidate for inclusion after all.

**Pull request closed without merging**